src/styles/List.module.css

```
@@ -5,9 +5,13 @@
 .list {
   background-color: #f7fdf6;
   border: 1px solid #53989c;
-  list-style: none;
   padding: 5px;
+  padding-left: 20px;
```

左だけじゃなくて右にもpaddingは入れたほうがいいかも
```suggestion
  padding: 5px 20px;
```